src/scripts/Controller.js

```diff
@@ -719,13 +733,57 @@ export default class SitemapController {
 		await this.store.logOut().finally(async () => await this.showAuthPage());
 	}
 
-	async showSitemaps() {
+	async showProjects() {
 		this.clearState();
-		this.setActiveNavigationButton('sitemaps');
+		this.setActiveNavigationButton('projects');
+
+		$(`#sitemaps-nav-button`).prop('disabled', true);
+		$('#create-sitemap-nav-button').prop('disabled', true);
+
+		const projects = await this.store.getAllProjects();
+
+		if (projects.error_msg) {
+			$('#sitemaps').hide();
+			$('#viewport').html(
+				'<div class="container"><div data-i18n="get_sitemap_error"></div></div>'
+			);
+			Translator.translatePage();
+		}
```

Comment:
I believe we don't send error_msg from storage, neither does gql api

Reply:
done